e3sm_diags/driver/lat_lon_driver.py

```diff
@@ -87,45 +91,20 @@ def run_diag(parameter):
             parameter.var_id = var
 
             mv1 = test_data.get_climo_variable(var, season)
-            mv2 = ref_data.get_climo_variable(var, season)
+            try:
+                mv2 = ref_data.get_climo_variable(var, season)
+            except Exception:
+                mv2 = mv1
```

Comment:
using "mv2=mv1" is a convenient way to have a fake datasets with all mv1 attributes and values..

Reply:
Just a note to say I've learned the hard way that a general `Exception` can be risky. It catches a `SyntaxError` in the code itself.

Reply:
Now use exception (RuntimeError, IOError) instead.